convertCSVtoJSON.js

```diff
@@ -3,6 +3,7 @@ const csv = require("csvtojson");
 
```

Comment:
Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.

Reply:
+4